README.md

```diff
@@ -147,6 +147,23 @@ The following platforms are currently not supported with the gProfiler executabl
 
 **Remark:** container-based execution works and can be used in those cases.
 
+## Running as systemd service
+
+You can generate a systemd service configuration that [runs gProfiler as an executable](#running-as-an-executable) (and therefore, bears the same [known issues](#executable-known-issues)) by running:
+
+``` bash
+GPROFILER_VERSION=<VERSION> GPROFILER_TOKEN=<TOKEN> GPROFILER_SERVICE=<SERVICE_NAME> ./deploy/systemd/create_systemd_service.sh
+```
+
+This script generates `granulate-gprofiler.service` in your working directory, and you can go ahead and install it by:
+```
+systemctl enable $(pwd)/granulate-gprofiler.service
```

Comment:
This copies the file to systemd system directories?

Reply:
Symlink

Reply:
Hm not ideal but let the user handleit